containers/scripts/crlite-generate.sh

```diff
@@ -57,18 +57,6 @@ ${workflow}/2-generate_mlbf ${ID} \
               --reason-set all \
               --filter-type cascade
 
-${workflow}/2-generate_mlbf ${ID} \
-              --filter-bucket ${crlite_filter_bucket:-crlite_filters_staging} \
-              --statsd-host ${statsdHost} \
-              --reason-set specified \
```

Comment:
If we're not going to publish "specified" or "priority" filters, should we remove their mention from `modules/libpref/init/StaticPrefList.yaml`?

Reply:
Yeah, I guess I could do that in D225261. They'll still be available until we cut a new crlite release and push it to prod, but I plan to do that pretty quickly.